src/components/Warning.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="warning">
+    <div class="container logo">
+      <img
+        class="brand-logo"
+        src="../assets/imgs/icons/signup/vsys_logo.svg">
+    </div>
+    <div class="image">
+      <div class="note"
+           :style="note">
+      </div>
+      <div class="title">
+        Warning
+      </div>
+      <div class="reason">
+        This may be caused by following reasons:
+      </div>
+      <div class="content">
+        <ul>
+          <li>Network is disconnected</li>
+          <li>Can not connect to the api server</li>
+          <li>Local javascript is crushed</li>
+        </ul>
+      </div>
+      <!--
+      <div class="title">
+        Warning
+      </div>
+      <div class="reason">
+        This may be caused by following reasons:
+      </div>
+      <div class="content">
+        <ul>
+          <li>Network is disconnected</li>
+          <li>Can not connect to the api server</li>
+          <li>Local javascript is crushed</li>
+        </ul>
+      </div>
+      -->
+    </div>
+  </div>
+</template>
+
+<script>
+import {NODE_IP} from '../constants'
+export default {
+    name: 'Warning',
+    data() {
+        return {
+            note: {
+                backgroundImage: 'url(' + require('../assets/imgs/icons/operate/img_warning.svg') + ')',
+                backgroundRepeat: 'no-repeat'
+            }
+        }
+    },
+    created() {
+        const url = NODE_IP + '/blocks/height'
+        this.$http.get(url).then(response => {
+            this.$router.push('/login')
+        }, response => {
+            this.$router.push('/warning')
+        })
+    }
+}
+</script>
+
+<style scoped>
+.warning{
+    width: 100%;
+    height: 100%;
+    background-color: #fafafa;
```

Comment:
Seeing from mobile, the left background is grey while right background is white. 
![p1](https://user-images.githubusercontent.com/41148850/51783662-2742eb80-2178-11e9-8dba-d5e58652ca82.jpeg)
Suggest change background-color to #FFFFFF here.